Projects/views.py

```diff
@@ -354,3 +334,32 @@ def change_label(request):
         issue.save()
         return HttpResponse(res)
     else: return HttpResponseBadRequest()
+
+def new_issue(request):
+    print('adding issue')
+    if request.method == 'POST':
+        if request.user.is_staff:
+            print('in add issue method')
+            ti = request.POST.get('title_issue')
+            li = request.POST.get('link_issue')
+            tp = request.POST.get('title_project')
+            lp = request.POST.get('link_project')
+            lev = int(request.POST.get('level'))
```

Comment:
what if mentor inputs a char instead of an int

Reply:
I thought about this but I thought that mentors, unlike normal user, would make no such mistakes. :sweat_smile:

Reply:
> Added model form to add issue also this model forms validates URL and numeric value for level and points of issue
> fixes #136

as I said above the integer fields are validated in form itself. So user cannot insert value other than an integer. One thing I have to update is to make all fields required in the form. So this would not make any problem
Will this be a correct solution?